spec/components/trainees/confirmation/course_details/view_preview.rb

```diff
@@ -24,6 +28,17 @@ def mock_trainee
             training_route: TRAINING_ROUTE_ENUMS[:assessment_only],
           )
         end
+
+        def mock_trainee_with_multiple_subjects
+          Trainee.new(
+            id: 1,
+            subject: "Primary",
+            subject_two: "Science",
+            course_age_range: [3, 11],
+            course_start_date: Date.new(2020, 0o1, 28),
```

Comment:
```suggestion
            course_start_date: Date.new(2020, 01, 28),
```

Reply:
Changed to be just 1 instead to appease rubocop.